### PR TITLE
Spots

### PR DESCRIPTION
**Pull request by AMBALERT1**
- [x] removed CreatedAt () & UpdatedAt() from the seed file 
- [x] removed onDelete and onUpdate from the model 
- [x] removed Associations from the spots model

### CODE REVIEW

File: backend/db/seeders/20240522125843-seed-spots.js [hunk 1]
```diff
@@ -14,11 +19,9 @@ module.exports = {
       lng: -122.4194,
       name: 'Wonderful Spot',
       description: 'A beautiful spot in the hear of the city.',
-      price: 100.00
-      previewImage: ,
-      avgRating: 5.5,
-      createdAt: new Date(),
-      updatedAt: new Date()
+      price: 100.00,
+      previewImage: images/image1.jpg,
```
Comment (AnthonyBronca) [inline backend/db/seeders/20240522125843-seed-spots.js]: Reference what this type of data is supposed to be in your db diagram

**Review by AnthonyBronca — CHANGES_REQUESTED** [review]

**Comment by AnthonyBronca** [comment]
> - [x] removed CreatedAt () & UpdatedAt() from the seed file 
> 
> - [ ] removed onDelete and onUpdate from the model 
> 
> - [ ] removed Associations from the spots model 
> 
>  

You mean you removed it from the migration, not the model?

File: backend/db/seeders/20240522125843-seed-spots.js (hunk 1 above)
Comment (AMBALERT1) [reply backend/db/seeders/20240522125843-seed-spots.js]: When I view my DB Diagram . its showing that this type of data should be datetime

Comment (AnthonyBronca) [reply backend/db/seeders/20240522125843-seed-spots.js]: Line 23? The "previewImage"?

Comment (AMBALERT1) [reply backend/db/seeders/20240522125843-seed-spots.js]: Looking at my db digram . previewimage is not under spots. But its showing via the API docs. So I would need to update my DB diagram to reflect

Comment (AnthonyBronca) [reply backend/db/seeders/20240522125843-seed-spots.js]: Ahh so you do not want to have `previewImage` in the spots seeder, migration, or the models. right?

Comment (AMBALERT1) [reply backend/db/seeders/20240522125843-seed-spots.js]: No I do want it, because based of the API docs it appears that its needed

Comment (AMBALERT1) [reply backend/db/seeders/20240522125843-seed-spots.js]: Nevermind! I think its best i remove it

**Review by AnthonyBronca — APPROVED** [review]

**Pull request merged**